release/models/aft/openconfig-aft-state-synced.yang

```diff
@@ -49,7 +55,7 @@ submodule openconfig-aft-state-synced {
 
       leaf ipv4-unicast {
         type boolean;
-        default false;
+        default true;
```

Comment:
I'm not sure that we should change this default -- if the default is true in some implementation, then can't it just send a value of `true` here? Otherwise we're assuming that the happy case has happened before /any/ telemetry as such has been sent, which seems a bad idea for a health signal.

Reply:
I think this sets a bad precedent. Suppose vendor X discovers an internal synchronization issue between actual state and telemetry state and they want to expose this issue (for good reasons) in state. Why should every other vendor have to report that they don't have this synchronization issue? With a properly chosen default, this can be avoided.

Reply:
It's completely legal to stream false, even when the schema default is false. I think this would cover your use case, yes @nokia1adam ?

Reply:
Hi @dplore , I don't think this helps me.

In the implementation I am working with, the false value represents a problem that requires investigation. So I certainly hope to be not streaming false all the time. The default of false requires me to constantly stream the state of this leaf as true, which is consuming extra resources just to indicate a normal/good state.

Reply:
I see the argument, but there are always going to be cases where such differences exist. In the overall volume of telemetry of AFT, does this one leaf cause us issues? Isn't it just sent once at the start of the session being set to true (with a `TARGET_DEFINED` or `ON_CHANGE` subscription)? If not, let's discuss with other implementations as to what the best choice here is.

(if we change the default to true, and some implementation says that until there's an explicit signal that it's false - then we just move the work around).

On this matter, from an operational perspective, assuming that things are healthy until they say unhealthy seems less preferable -- it seems like it's better to have some explicit signal that it's healthy.